src/utils/fetchMarkdown.ts

```diff
@@ -0,0 +1,18 @@
+interface MarkdownResult {
```

Comment:
nit, would prefer this instead of `export type ...` at the bottom
```suggestion
export interface MarkdownResult {
```